src/EventRegistry.php

```diff
@@ -160,4 +161,13 @@ public function getEventClassName(string $event)
 
         return null;
     }
+
+    public function listActiveEvents(array $events)
+    {
+        foreach ($events as &$event) {
```

Comment:
I'd prefer to avoid a reference when it's reasonably possible

Reply:
NP! refactored